client/.eslintrc.json

```diff
@@ -11,7 +11,7 @@
     "ecmaFeatures": {
       "jsx": true
     },
-    "project": "./client/tsconfig.json"
```

Comment:
@ejwoods  do you by chance remember from Friday why we needed this part of path here?  I thought we needed this for a reason, but everything seems to check out without it.

Reply:
This was added while testing different solutions. I believe this would cause eslint to look for a `client/client/` directory, which doesn't exist. `"project": "./tsconfig.json"` should be the correct path for this file.